.github/workflows/xamarin.yml

```diff
@@ -62,11 +62,25 @@ jobs:
 
 
   iOS:
-    runs-on: macos-10.15
+    runs-on: macos-latest
     steps:
     - uses: actions/checkout@v2
       with:
         fetch-depth: 0
+    - name: Set default VS version
+      run: |
+        mv "/Applications/Visual Studio.app" "/Applications/Visual Studio 2022.app"
+        mv "/Applications/Visual Studio 2019.app" "/Applications/Visual Studio.app"
+
+      # macOS agents recently have empty NuGet config files, resulting in restore failures,
+      # see https://github.com/actions/virtual-environments/issues/5768
+      # Add the global nuget package source manually for now.
+    - name: Setup NuGet.Config
```

Comment:
Maybe copy the existing nuget file that's already in the repo instead of creating a new one?

Reply:
done